README.md

```diff
@@ -4,44 +4,27 @@
 [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
 # Project Calico
 
-Project Calico provides
-
-- A simple, pure layer 3 networking approach with no overlays for networking
-  "workloads" such as VMs and containers.
-- A distributed firewall implementing rich and flexible network policy,
-  imposed at ingress/egress to each workload.
-
-For more information see [the Project Calico website](http://www.projectcalico.org/learn/).
+<blockquote>
```

Comment:
HTML fragments in markdown is an officially supported thing.  The HTML fragment proposed gives a better formatted notice for our purposes I think.